sernet.gs.service/src/sernet/verinice/model/common/CnATreeElement.java

```diff
@@ -69,203 +75,201 @@ private Logger getLog() {
         }
         return log;
     }
-    
-	private Integer dbId;
-	
-	private Integer scopeId;
-	
-	private String extId;
-
-	private String sourceId;
-	
-	private String objectType;
-	
-	private String iconPath;
-	
-	public int getNumericProperty(String propertyTypeId) {
-	    PropertyList properties = getEntity().getProperties(propertyTypeId);
-	    if (properties == null || properties.getProperties().size()==0){
-	        return 0;
-	    } else {
-	        return properties.getProperty(0).getNumericPropertyValue();
-	    }
-	}
-	
-	public void setNumericProperty(String propTypeId, int value) {
-	    EntityType type = getTypeFactory().getEntityType(getEntity().getEntityType());
+
```

Comment:
This diff is pretty useless, because there are a lot of line break and white space changes and that's why changes are not comprehensible.